src/components/activity-page/bottom-buttons.scss

```diff
@@ -9,4 +9,10 @@
     justify-content: center;
   }
 
+  button:disabled {
+    background-color: $cc-teal-light2;
```

Comment:
We probably want to add `cursor: auto;` here so disabled buttons don't look clickable.

Reply:
Also this color needs to be derived from the primary theme colors.  Otherwise, when a theme color is set, the other buttons will change color and these will remain teal.  Add the themeButtons URL parameter and change a color to see what I mean.